examples/rendezvous/src/bin/rzv-identify.rs

```diff
@@ -19,48 +19,47 @@
 // DEALINGS IN THE SOFTWARE.
 
 use futures::StreamExt;
-use libp2p_core::{identity, upgrade::Version, Multiaddr, PeerId, Transport};
-use libp2p_identify as identify;
-use libp2p_ping as ping;
-use libp2p_rendezvous as rendezvous;
-use libp2p_swarm::{keep_alive, NetworkBehaviour, Swarm, SwarmEvent};
+use libp2p::{
+    core::transport::upgrade::Version,
+    identify, identity, ping, rendezvous,
+    swarm::{keep_alive, NetworkBehaviour, Swarm, SwarmEvent},
+    Multiaddr, PeerId, Transport,
+};
 use std::time::Duration;
-use void::Void;
 
 #[tokio::main]
 async fn main() {
     env_logger::init();
 
+    let key_pair = identity::Keypair::generate_ed25519();
     let rendezvous_point_address = "/ip4/127.0.0.1/tcp/62649".parse::<Multiaddr>().unwrap();
     let rendezvous_point = "12D3KooWDpJ7As7BWAwRMfu1VU2WCqNjvq387JEYKDBj4kx6nXTN"
         .parse()
         .unwrap();
 
-    let identity = identity::Keypair::generate_ed25519();
```

Comment:
Did we have to move and rename that? In general I'd like to keep diffs as small as possible.

Reply:
No, we don't, but it is a part of the unification of the Rendezvous examples, makes it clear what is the same and what is different in them. Previously keypair was generated in a random line in the code, now it sticks to the beginning like in any other example.

Reply:
Got it, thanks!